.github/workflows/kind-e2e.yaml

```diff
@@ -25,9 +25,12 @@ jobs:
 
         gateway:
         - quay.io/maistra-dev/proxyv2-ubi8:2.4-latest
-        - docker.io/envoyproxy/envoy:v1.26-latest
-        - docker.io/envoyproxy/envoy:v1.27-latest
-        - docker.io/envoyproxy/envoy:v1.28-latest
+        - docker.io/envoyproxy/envoy:distroless-v1.26-latest
+        - docker.io/envoyproxy/envoy:distroless-v1.27-latest
+        - docker.io/envoyproxy/envoy:distroless-v1.28-latest
```

Comment:
It seems that < 1.28 are EOL see https://github.com/envoyproxy/envoy/blob/main/RELEASES.md#major-release-schedule. Probably good to start with 1.28?

Reply:
Fixed